### ISSUE TITLE
Revisit each stage of the getting started process

### ISSUE THREAD

**Issue by siyuan0322** [alibaba/GraphScope#742]
Many users still encounter many difficulties when experiencing GraphScope, we need to revisit each stage carefully to further lower the getting started threshold, some thoughts include:
1. Investigate time spent every stage when launching session,
2. Provide detailed guide to deploy locally,
3. Give loading graph code template for users to fill in,
4. The UDF apps guide is still a weakness, 
5. Provide more examples targeting at different user scenarios,
6. The GraphScope Store should also be mentioned and have an e2e guide itself,
7. More system OS, version adaption. 

In short, we want to make users face fewer errors when they are going through the basic procedures.
*BTW, the notebook tutorials is already lagged behind.*

**Comment by siyuan0322** [comment]
Closed as 1, 2, 6, 7 have already been implemented in previous commit, and 3, 5 moved to a new issue #1079, and 4 are in low priority now.
